handlers/propertyInspectionReports.js

```diff
@@ -27,5 +33,31 @@ function propertyInspectionReports(request, reply) {
 
 }
 
-module.exports = propertyInspectionReports;
+function postHandler(request, reply) {
+  var payload = request.payload;
+  var userId = 6; //Temporary hardcoded - userid currently can't be passed via params
```

Comment:
If the user is authed there will be an auth.artifacts object. What gets put
on that though is up to you. I set the user model when they are
successfully authed which is why we can grab their id. See hapi docs for
the specifics.
On 22 Sep 2015 5:44 pm, "PeterHsieh" <notifications@github.com> wrote:

> In handlers/propertyInspectionReports.js
> <https://github.com/nirrek/wallflyServer/pull/8#discussion_r40059652>:
>
> > @@ -27,5 +33,31 @@ function propertyInspectionReports(request, reply) {
> >
> >  }
> >
> > -module.exports = propertyInspectionReports;
> > +function postHandler(request, reply) {
> > +  var payload = request.payload;
> > +  var userId = 6; //Temporary hardcoded - userid currently can't be passed via params
>
> Ahh forgot to ask you about that! Yeah I wasn't sure how to retrieve the
> userId at the time as it was different from the previous implementations. I
> was in a rush so I had overlooked it. Thanks again for the reminder! I'll
> try make sure it doesn't happen again (and wait to ask before doing a PR).
>
> I saw the fix using auth.artifacts, curious when this is available for use?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nirrek/wallflyServer/pull/8/files#r40059652>.
>